ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -873,6 +882,34 @@ protected void maybeProcessCallbacks() {
       }
     }
 
+    /**
+     * Decompress the compressed buffer specified.
+     * If the buffer is not compressed, based on blobCompressed property, then return the input buffer.
+     *
+     * @param sourceBuffer The source buffer to decompress.
+     * @return New buffer if decompressed successfully and the source buffer is released.
+     *         Original buffer is blob not compressed.  NULL if failed and source buffer is released.
+     */
+    protected ByteBuf decompressContent(ByteBuf sourceBuffer) {
+      if (!isChunkCompressed) {
+        logger.info("Blob is not compressed.  Return source buffer.");
```

Comment:
This will overwhelm the log, plus its not very informative. It doesn't print out blob id

Reply:
I'll remove this log in next revision.